lib/recurly/pager.php

```diff
@@ -101,9 +100,9 @@ protected function _loadFrom($uri, $params = null) {
   }
 
   protected function _afterParseResponse($response, $uri) {
-    $this->_href = $uri;
     $this->_loadRecordCount($response);
     $this->_loadLinks($response);
+    $this->_href = isset($this->_links['start']) ? $this->_links['start'] : $uri;
```

Comment:
This fixes a bug where we were rewinding to the second page rather than the first.